pkg/disruption/backend/sampler/remote.go

```diff
@@ -335,8 +324,8 @@ func createNamespace(ctx context.Context, clientset *kubernetes.Clientset) error
 
 	client := clientset.CoreV1().Namespaces()
 	actualNamespace, err := client.Create(ctx, namespaceObj, metav1.CreateOptions{})
-	if err != nil && !apierrors.IsAlreadyExists(err) {
-		return fmt.Errorf("error creating namespace: %v", err)
+	if err != nil {
```

Comment:
this change is still good. Generated names should just about never alreadyexist.